### PR TITLE
Fix Denmark - re-add Christmas Eve

### PR DESCRIPTION
**Pull request by KidkArolis**
I've previously removed Christmas Eve from Denmark (together with some other, actually inaccurate public holidays), but as have been pointed out to me by our Danish customers - Christmas Eve is actually being treated as public holiday (even if it's legally not recognized).

Extra sources:
- https://www.officeholidays.com/countries/denmark
- https://www.feiertagskalender.ch/index.php?geo=3289&klasse=0&hl=en

In conclusion, while this stuff is always gonna be ambiguos - I think this particular setup for Denmark is a better default that more accurately represents how people deal with the public holidays there.

### CODE REVIEW

**Comment by brunobord** [comment]
right. taking this over. I'll make a release right after

**Comment by brunobord** [comment]
Merged in https://github.com/peopledoc/workalendar/commit/6878077bf61261291491ad6a0d29a64d876eeade

**Comment by brunobord** [comment]
version 7.1.0 was released and published on PyPI including your fix to the Denmark calendar.
Happy upgrade!
